### ISSUE TITLE
Reactivating after doing the upgrade...

### ISSUE THREAD

**Issue by AlainMelsens** [burlingtonbytes/wp-blockade#2]
Dear,
Is it normal that the "wp-blockade" plugin will be also deactivated after doing an upgrade?
After I have done the new upgrade, the "wp-blockade" plugin was deactivated. I must reactivate the "wp-blockade" plugin and than, everything was working as before.
Normally, all WordPress plugins that are running (= activated) and there are upgrades, they stay also activated after upgrades are done.
This is simply a little issue, not really a big problem nor an error.
Thanks in advance to verify and confirm this.
Best Regards,
Alain Melsens

**Comment by gschoppe** [comment]
It is uncommon for a plugin to deactivate on upgrade, but it can happen.  Sometimes the upgrade process doesn't complete correctly, and wordpress doesn't finish the last step.  I haven't yet been able to reproduce the issue with any of my test installs, so it may just be a fluke, but if you have the time, could you tell me what other plugins are running on your install?

**Comment by AlainMelsens** [comment]
Hello,
Ok, I understand. Anyway, the upgrade is working well. No problem.
With more time, I can try to restore an older backup on an other virtual website with less plugins or with plugins deactivated before doing the upgrade and so I can see what's happening.
I will keep you informed, but maybe I can do this, this weekend. 
Regards,
Alain Melsens

**Comment by AlainMelsens** [comment]
Hello,
I have tried it to make an upgrade on a new WordPress website without any other plugins installed and activated.
This time, the upgrade is done without deactivating the "wp-blockade" plugin as common.
So, you are right, it should be an other plugin who was interrupting maybe the previous upgrade on the other website. 
So, no problem, I will check lateron wich plugin has done the issue and will keep you informed.
Regards,
Alain Melsens

**Comment by AlainMelsens** [comment]
Hello,
I have retried the same situation on the same website. So, before I have deactivated the "wp-blockade" plugin and deleted also. Then reinstalled the older version 0.9.4 and activated also, and then, I have done the upgrade and now it's working as normal. The "wp-blockade" plugin was still activated afer done the upgrade.
Maybe it was a temporarily problem.
So, I can close this issue.
My apologies for this unnecessary intervention.
Best Regards,
Alain Melsens
